### PR TITLE
Sgreenberg feb19

### PR DESCRIPTION
**Pull request by sgreenberg**
Few changes based on my experience getting the db set up for initial use. 

Note that I believe explicitly saving the voting_rights csv as UTF-8 should work fine across Windows/OSX, but it might be good for someone with a Windows machine to double check that if possible. 

Thanks so much!

### CODE REVIEW

File: SD_DB_Setup.sql [hunk 1]
```diff
@@ -1,4 +1,5 @@
-drop database SisterDistrict_dev;
+SET SQL_MODE='ALLOW_INVALID_DATES';
+drop database if exists SisterDistrict_dev;
```
Comment (nsudarsanam) [inline SD_DB_Setup.sql:3]: @LinneaHarts has fixed the date format for the expenditure date column, is that why this is here? if not, wanted to double-check this change.

File: voting_rights/voting_rights_2016.csv [hunk 2]
```diff
@@ -76,12 +76,12 @@ TN,2016,1,Strict photo ID,"At polling places in Tennessee, voters must present g
...
 WA,2016,1,Non-strict non-photo ID,"Washington is a vote-by-mail state. County auditors are required to open at least one vote center where voters can cast a ballot in person, and a valid photo ID is required.",No if voting in person but most people vote by mail.,No pending legislation.
 WV,2016,0,Non-strict non-photo ID; goes into effect in 2018,First-time voters in West Virginia who registered by mail must present identification. Valid identification includes photo and non-photo identification.,Yes unless you are a first-time voter.,There was legislative movement in 2016. Nothing new appears to be in the news.
 WI,2016,1,Strict photo ID ,Voters in Wisconsin are required to present photo identification at the polls.,No.,"July 2016 in Wisconsin, a federal judge ruled that voters without ID must still be allowed to vote, saying that a state law, ostensibly intended to prevent voter fraud, led instead to disenfranchisement. But a footnote for all ' he three states that sunk Clinton: Pennsylvania, Michigan, and
```
Comment (nsudarsanam) [inline voting_rights/voting_rights_2016.csv]: @augie can you re-run your script with the csv from git? we want to make sure the csv is still working with your script..thank you!

**Review by nsudarsanam — COMMENTED** [review]
I've added a few questions..but its a bit confusing because it looks like the change was merged? Checking with @LinneaHarts

File: SD_DB_Setup.sql (hunk 1 above)
Comment (LinneaHarts) [reply SD_DB_Setup.sql:3]: Yes, I fixed the date format.

Comment (sgreenberg) [reply SD_DB_Setup.sql:3]: Okay. So we can remove 'SET SQL_MODE='ALLOW_INVALID_DATES';' ? 

I can do that tonight. Sorry. I should have asked about that.

File: voting_rights/voting_rights_2016.csv (hunk 2 above)
Comment (sgreenberg) [reply voting_rights/voting_rights_2016.csv]: @augie - Are you using a Windows machine? I had to explicitly save the CSV out as UTF-8 in order to get voting_rights.py to work on my Mac. Nisha's question is a good one. We should make sure that it still works correctly on your machine.

Comment (sgreenberg) [reply voting_rights/voting_rights_2016.csv]: And thanks @nsudarsanam  for following up on this.

File: SD_DB_Setup.sql (hunk 1 above)
Comment (LinneaHarts) [reply SD_DB_Setup.sql:3]: I did it while resolving conflicts. It's all gone now.

**Pull request merged**